src/CommunityVoices/App/Api/View/Quote.php

```diff
@@ -81,6 +81,19 @@ protected function postQuoteUpload()
         return $response;
     }
 
+    protected function postBatchDraft($request, $errors)
+    {
+        list($sheetData, $columnNameWarnings, $columnNameErrors, $unpairedQuotes, $validIdentifiers) = $errors;
```

Comment:
Can we not name the second argument `$errors` if it is really not holding the errors in any way?  I believe how the Dispatcher is set up, you can name it any way.

Reply:
@Sammidysam Noted.

Reply:
Fixed